cypress/integration/models/jobfunctions.ts

```diff
@@ -1,12 +1,12 @@
 import { controls, jobfunctions, tdTag, trTag } from "../types/constants";
 import { navMenu, navTab } from "../views/menu.view";
 import { jobfunctionNameInput } from "../views/jobfunctions.view";
-import { confirmButton, editButton, deleteButton } from "../views/commoncontrols.view";
+import * as commonView from "../views/commoncontrols.view";
 import { clickByText, inputText, click, selectItemsPerPage, submitForm } from "../../utils/utils";
 import * as faker from "faker";
 
 export class Jobfunctions {
-    jobfunctionName;
+    jobfunctionName= faker.name.jobType();
```

Comment:
Can we have this in a function? It will be fine if we dont set class variable. Calling faker() again and again in test file is somewhat tedious.